### ISSUE TITLE
Issues around Tanzu cli login command

### ISSUE THREAD

**Issue by thesteve0** [vmware-tanzu/tanzu-framework#1859]

**Comment by thesteve0** [comment]
It also turns out that the --name has to match the name of the cluster. I am not sure what "name" we need to match but my guess is the actual  cluster name from the context. But I am confused why we need that when the information is already in the kube config file

**Comment by codegold79** [comment]
I agree the help messages should be more helpful, so thank you, @thesteve0, for submitting this issue.

> It also turns out that the --name has to match the name of the cluster.
Is the error the same as what was reported in #1859?
